content/blog/snowflake-provider-launch/index.md

```diff
@@ -29,5 +29,5 @@ Pulumi is bringing management of your Snowflake resources into the same place as
 - Create a service user and push the key into the secrets manager of your choice
 - Rotate keys
 
-You can check out the details in the official [Snowflake Provider documentation
-](/registry/packages/snowflake/api-docs/provider/), with references for Python, Go, TypeScript, and C#!
+You can check out the details in the official [Snowflake Provider documentation](/registry/packages/snowflake/api-docs/provider/), with
```

Comment:
This fixes a rendering issue in the link. The current link looks like this (note the extra ` `):

<img width="748" alt="Screenshot 2025-02-11 at 2 44 00 PM" src="https://github.com/user-attachments/assets/1ce89f38-e061-403f-b715-d8207382a4fc" />